database/database_manager.py

```diff
@@ -0,0 +1,93 @@
+"""
+database_manager.py - Debug and diagnostics commands
+Provides postgreSQL connectivity for mechasqueak3.
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+Licensed under the BSD 3-Clause License.
+See LICENSE.md
+"""
+from config import config
+from psycopg2 import sql, pool
+from utils.ratlib import Singleton
+import logging
+import psycopg2
+
+log = logging.getLogger(f"mecha.{__name__}")
+
+
+class DatabaseManager(Singleton, object):
```

Comment:
After long discussion, i don't believe inheriting from `Singleton` is a good idea.

Reply:
This will be fixed in another iterative pass, when I move the DBM module to a context manager.